### PR TITLE
Updated Readme File with instructions for Windows

### PR DESCRIPTION
**Pull request by baskarmib**
Updated the Readme file with install instructions for Windows Machine.

### CODE REVIEW

**Comment by baskarmib** [comment]
This Pull request also has the changes for the Home Screen provided in the latest design.

File: README.md [hunk 1]
```diff
@@ -32,3 +32,28 @@ Follow these steps to start your dev environment:
...
+
+`$ make setup-environment
+docker-compose exec --user=application web yarn install
+the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'
+make: *** [Makefile:164: install] Error 1`
+
+
+To fix this we can modify the Makefile in build folder as shown below.
+
+- DOCKER_WEB_EXEC=winpty docker-compose exec --user=application web
```
Comment (BradGriffith) [inline README.md:57]: These modified lines may actually be better in the env.mk file that doesn't get committed to the repo. Put in that file, they would just override what is in Makefile. That would help ensure a developer on Windows doesn't accidentally commit the change for everyone. I'll go ahead and merge this, but feel free to make that small change to the README.md later.

File: src/resources/js/Pages/Welcome.vue [hunk 2]
```diff
@@ -3,30 +3,26 @@
 
         <div id="page-wrapper" class="h-screen w-full flex flex-wrap">
             <div class="bg-blue w-full text-center text-white p-1">
-                &nbsp;
+               Important Covid-19 Information. <a href="https://coronavirus.ohio.gov/wps/portal/gov/covid-19/home">Learn More.</a>
```
Comment (BradGriffith) [inline src/resources/js/Pages/Welcome.vue]: They decided late in the game to not have this line in the header, so I'll take it out.

File: src/resources/js/Pages/Welcome.vue [hunk 3]
```diff
@@ -3,30 +3,26 @@
...
             </div>
             <div class="border-b border-blue-500 w-full p-1">
                 <h1 class="logo float-left text-center text-2xl mx-auto my-1 ml-2">Vaccinate OH</h1>
 
                 <div class="float-right px-6 py-2 z-40">
+                    <inertia-link href="/" class="text-blue font-bold ml-4">Home</inertia-link>
 
-                    <inertia-link href="/dashboard" class="text-sm text-gray-700 underline ml-4">Stats</inertia-link>
+                    <inertia-link href="https://info.vaccinateoh.org" class="text-blue font-bold ml-4">About Us</inertia-link>
```
Comment (BradGriffith) [inline src/resources/js/Pages/Welcome.vue]: I'm going to change these from inertia-link to regular old a tags. Inertia loads links without a page reload, which isn't really necessary. I know you were just following what was here already.

File: src/resources/js/Pages/Welcome.vue [hunk 4]
```diff
@@ -178,7 +195,7 @@ export default {
                 markers: [
                 ],
             },
-            search_q: '43210',
+            search_q: 'Zipcode',
```
Comment (BradGriffith) [inline src/resources/js/Pages/Welcome.vue]: I don't think we want to populate this value at all initially. I'll remove this default value.

**Review by BradGriffith — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (baskarmib) [reply README.md:57]: Sure I will update the ReadMe file with changes.

**Pull request merged**